packages/pocket-ic/src/common/rest.rs

```diff
@@ -766,27 +780,21 @@ impl ExtendedSubnetConfigSet {
             (sns, "sns"),
             (nns_ui, "nns_ui"),
         ] {
-            // using `EmptyConfig { }` explicitly
-            // to force an update after adding a new field to `EmptyConfig`
-            if let Some(EmptyConfig {}) = flag {
+            if flag.is_some() {
```

Comment:
I would rather keep a more explicit logic:
```suggestion
            if flag.unwrap_or_default() == IcpFeaturesConfig::DefaultConfig {
```

Reply:
If we add a new variant to `IcpFeaturesConfig`, then the suggested equality might not hold if we change the type `IcpFeaturesConfig`, but we want the subsequent check to apply independently of how the feature is configured (we want to ensure the subnet is empty no matter how we initialize the corresponding system canister).

Reply:
Makes sense, thanks for the clarification